src/lib/coda_transition/external_transition.ml

```diff
@@ -1069,6 +1076,10 @@ module Staged_ledger_validation = struct
     in
     let staged_ledger_diff = staged_ledger_diff transition in
     let apply_start_time = Core.Time.now () in
+    let coinbase_receiver =
```

Comment:
REVIEWERS: does this use of the parent protocol state make sense?

Reply:
I think this takes the coinbase receiver from the previous block, which isn't what we want. This should use the transition's protocol state, i.e. `coinbase_receiver transition` using the accessor added above.